sdk/daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SBuiltinFun.scala

```diff
@@ -697,6 +698,42 @@ private[lf] object SBuiltinFun {
     }
   }
 
+  final case object SBDecodeHex extends SBuiltinFun(1) {
+    override private[speedy] def execute[Q](
+        args: util.ArrayList[SValue],
+        machine: Machine[Q],
+    ): Control[Q] = {
+      try {
+        val hexArg = Ref.HexString.assertFromString(getSText(args, 0))
+        val arg = new String(Ref.HexString.decode(hexArg).toByteArray, StandardCharsets.UTF_8)
```

Comment:
```suggestion
        val arg = Ref.HexString.decode(hexArg).toByteString.toStringUtf8
```
Maybe we could add `toStringUtf8: String` to `com.data.Bytes`